frontend/packages/ui/src/fonts/fonts.less

```diff
@@ -1,2 +1,3 @@
-@import 'Manrope/Manrope.less';
-@import 'OpenSans/OpenSans.less';
+@import 'NunitoSans/style.css';
```

Comment:
check if this is working when you are importing .less file, instead of .css

Reply:
it's not working :(